app/scripts/views/reset_password.js

```diff
@@ -11,14 +11,17 @@ define(function (require, exports, module) {
   const FlowEventsMixin = require('./mixins/flow-events-mixin');
   const FormView = require('./form');
   const PasswordResetMixin = require('./mixins/password-reset-mixin');
-  const preventDefaultThen = require('./base').preventDefaultThen;
   const ServiceMixin = require('./mixins/service-mixin');
   const Session = require('../lib/session');
   const Template = require('templates/reset_password.mustache');
 
   const t = (msg) => msg;
 
-  class ResetPasswordView extends FormView {
+  const ResetPasswordView = FormView.extend({
```

Comment:
I could probably also have done this without ditching `class` I guess, by setting `events` on `ResetPasswordView.prototype` instead. I opted to do it this way to be consistent with the other views, but I'm happy to change it if we'd prefer to gradually move in the direction of `class` syntax.

Reply:
I've been trying to figure out how to handle adding events and class syntax. Over in #6273 I have added the [transform-class-properties](https://github.com/mozilla/fxa-content-server/pull/6273/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR113) babel plugin which allows us to use the new [class properties syntax](https://babeljs.io/docs/en/babel-plugin-transform-class-properties/).

Using class properties we _could_ do something like:

```js
class ResetPasswordView extends FormView {
  events = {
    event_name: 'function_name'
  };

  constructor (options) {
    super(options);
    ...
  }
  ...
}
```

That's decent looking but I suspect it's broken with mixins semantically that compiles down to something similar to what you are fixing here. The way the class properties work, they are not set on the prototype as one would expect, rather the constructor is called and THEN the class properties are set, just like the version being fixed. They are not "class properties" but rather "instance properties". This implies everything will be broken using class properties if `events` are needed.

I'm sure there is some hackery to get around that, I just haven't figured it out yet.